sequencer.c

```diff
@@ -2032,6 +2039,20 @@ static int pick_commits(struct todo_list *todo_list, struct replay_opts *opts)
 			*end_of_arg = '\0';
 			res = do_exec(item->arg);
 			*end_of_arg = saved;
+
+                        /* Reread the todo file if it has changed. */
+                        if (res == 0) {
+                                struct stat st;
```

Comment:
It is a matter of taste, but I really like to keep indentation small. I would move this `struct stat st` declaration just after `int saved = *end_of_arg;` to save on one indentation level (especially given that we expect to need that variable, because we expect the `exec` command to succeed most of the time). As I said, it is a matter of taste, and your taste may be different.